integration/base/src/main/java/com/inrupt/client/integration/base/AccessGrantScenarios.java

```diff
@@ -479,6 +449,10 @@ void accessGrantGetRdfTest(final Session resourceOwnerSession, final Session req
         final SolidSyncClient resourceOwnerClient = SolidSyncClient.getClientBuilder()
             .build().session(resourceOwnerSession);
 
+        final URI testRDFresourceURI = URIBuilder.newBuilder(privateContainerURI)
+                .path("resource-accessGrantGetRdfTest.ttl")
+                .build();
+
         try (final SolidRDFSource resource = new SolidRDFSource(testRDFresourceURI, null, null)) {
             assertDoesNotThrow(() -> resourceOwnerClient.create(resource));
```

Comment:
A pattern I typically use with the JS client for such case, where the resource URI is irrelevant to the client (but still needs to be tracked) is to get the server to create the resource by POSTing to the parent container, which URL the client does care about (here that would be `privateContainerURI`). Is this a pattern that would be possible with the JCL, or does the target URL needs to be known before creating a resource?

Reply:
In the high-level client, we exclusively use PUT for create and update. POST and PATCH are usable directly in the low-level client.

Reply:
It's absolutely not critical for GA, but would something like `createChild` make sense in the `SolidClient` ?